code/modules/holodeck/holo_effect.dm

```diff
@@ -23,42 +23,15 @@
 /obj/effect/holodeck_effect/proc/safety(active)
 	return
 
-
 // Generates a holodeck-tracked card deck
 /obj/effect/holodeck_effect/cards
-	icon = 'icons/obj/toy.dmi'
-	icon_state = "deck_nanotrasen_full"
-	var/obj/item/toy/cards/deck/deck
-
-/obj/effect/holodeck_effect/cards/activate(obj/machinery/computer/holodeck/HC)
-	deck = new(loc)
-	safety(!(HC.obj_flags & EMAGGED))
-	deck.holo = HC
-	RegisterSignal(deck, COMSIG_PARENT_QDELETING, .proc/handle_card_delete)
-	return deck
-
-/obj/effect/holodeck_effect/cards/proc/handle_card_delete(datum/source)
-	SIGNAL_HANDLER
-	deck = null
-
-/obj/effect/holodeck_effect/cards/safety(active)
-	if(!deck)
-		return
-	if(active)
-		deck.card_hitsound = null
-		deck.card_force = 0
-		deck.card_throwforce = 0
-		deck.card_throw_speed = 3
-		deck.card_throw_range = 7
-		deck.card_attack_verb_continuous = string_list(list("attacks"))
-	else
-		deck.card_hitsound = 'sound/weapons/bladeslice.ogg'
-		deck.card_force = 5
-		deck.card_throwforce = 10
-		deck.card_throw_speed = 3
-		deck.card_throw_range = 7
-		deck.card_attack_verb_continuous = string_list(list("attacks", "slices", "dices", "slashes", "cuts"))
+	icon = 'icons/obj/playing_cards.dmi'
+	icon_state = "deck_syndicate_full"
 
+/obj/effect/holodeck_effect/cards/activate(obj/machinery/computer/holodeck/holodeck)
+	var/obj/item/toy/cards/deck/syndicate/holographic/deck = new(loc, holodeck)
+	deck.flags_1 |= HOLOGRAM_1
```

Comment:
holodeck already adds this flag to the atom returned by activate()

Reply:
edit: no it doesnt, thats an issue with the holodeck

Reply:
https://github.com/tgstation/tgstation/blob/47c8a5c6070f359773de1febdb773cf36f7f5c44/code/modules/holodeck/holo_effect.dm#L7-L17

I don't see where it's doing that?  The rest of the effects are using `flags_1 |= HOLOGRAM_1` to spawn their items.  I think this is because it's like a random spawner where they want to delete the effect, so we have to manually add the `HOLOGRAM_1` flag to any objects spawned.